autopush/web/webpush.py

```diff
@@ -62,7 +64,7 @@ def _router_completed(self, response, uaid_data, warning=""):
                 # be deleted. There is no longer valid route information for
                 # this record.
                 d = deferToThread(self.ap_settings.router.drop_user,
-                                  self.uaid)
+                                  uaid_data.get("uaid"))
```

Comment:
This still leaves open the possibility that the router might remove the uaid, why not just grab uaid directly and risk the key error? If we see a KeyError, then we can fix the router error.